doc/release/upcoming_changes/20083.new_feature.rst

```diff
@@ -0,0 +1,16 @@
+``hex`` and ``fromhex`` is now available to `numpy.floating`
+------------------------------------------------------------
+The float types have a new instance method :meth:`float.hex` and a
+corresponding new class method :meth:`float.fromhex` to convert
+floating-point numbers to and from hexadecimal strings,
+respectively.
+
+For more details on the methods see python docs on
+`float.hex <https://docs.python.org/3/library/stdtypes.html#float.hex>`__ and
+`float.fromhex <https://docs.python.org/3/library/stdtypes.html#float.fromhex>`__
+
+.. code-block:: python
```

Comment:
Could be mistaken, but I don't think `.. code-block` renders correctly without an empty line.
```suggestion
.. code-block:: python

```

Reply:
Ah my bad, will fix this. This unfortunately does not render in docs build making it a bit hard.

Reply:
Fixed in 34593e7